src/main/resources/META-INF/openapi.yml

```diff
@@ -0,0 +1,89 @@
+paths:
```

Comment:
So API documentation must be statically written along side with code, there's no way to annotate the code (for example as in .netcore or springboot) or programmatically provide the schema (for example as in fastify)?

Reply:
From what I've seen, annotations can only be used in `Helidon MP`, which is a slightly different framework. In this [page](https://helidon.io/docs/latest/index.html#/se/openapi/01_openapi) all options for `Helidon SE` are described, but I think the best way is to include the static file `META-INF/openapi.yml` as a configuration.